app/src/main/java/com/github/fribourgsdp/radio/game/handler/HostGameHandler.kt

```diff
@@ -101,6 +112,19 @@ class HostGameHandler(
         }
     }
 
+    private fun assignSong(songName : String){
+        db.updateCurrentSongOfGame(game.id, songName, singerDuration)
+            .addOnFailureListener {
+                Log.e( "HostGameHandler Error" , it.message.toString())
```

Comment:
You forgot to give the error to the function 😅 If you don't give it we won't be able to know what is the trace of the error. You can simply copy past what I put in the previous comment 👍🏻

Reply:
I put his here:

```kotlin
Log.e("HostGameHandler Error", "Song assignment: ${it.message}", it)
```